addons/isonline/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "IsOnline",
+  "description": "Adds a button to the bottom-right of the project page to set the thumbnail of your project to any image (including gifs).",
```

Comment:
I don't think this description is correct.

Also, does this have an effect on the project page? If not, you should remove the `projectPage` tag.

Reply:
It's just the description and tags from the thumbnail setter addon. This should probbably be a draft.

Reply:
yeah that was me copying an existing addon.json because i was lazy